.ruby-gemset

```diff
@@ -0,0 +1 @@
+gfsc-place-cal
```

Comment:
Is this likely to be applicable to other people's configurations? :)

Reply:
yeah let's not specify environment, we can add to `.gitignore` if needs be

Reply:
rbenv has a plugin allowing you a per-project set of gems (https://github.com/jf/rbenv-gemset). I find it keeps my dev environment lightweight and allows me to have multiple projects with different versions of gems at the same time. You can ignore and it won't interfere with anything, but it is super useful

Reply:
oh i hear you and this sounds good. just as project maintainers we don't really want to be in the business of telling people what their local development environment should be. dotfiles have a way of piling up you know? i think if you are really keen on putting this in the codebase then we should be adding stuff to the README etc about our suggested environment. maybe its fine for now tho as long as we have a note to do this later. altho pls remove space :) i.e. `gfsc-placecal`

Reply:
I see your point. dotfiles Is there a formal process for suggesting technologies? Like if we need to set up a background job system and needed to add redis to our dev/prod machines.

I checked the README and it doesn't mention anything about requirements at all. I could put something in right now to cover rbenv, rbenv-ruby ?

Reply:
yeah this is a great example of something that sounds great _one day_ but we truly do not need any time soon :)

you're welcome to file an issue and label it "code smell" or "tech debt" tho!